README.md

```diff
@@ -1,3 +1,41 @@
-# Offcial Ziptastic Java Library
-
-## Coming soon!
+# **Offcial Ziptastic Java Library**

```

Comment:
Would, Official Ziptastic Java Library (ALPHA) work just as well? I feel as if using BETA indicates more functionality/testing then is present at the moment.